app/views/posts/index.html.erb

```diff
@@ -124,8 +124,8 @@
                     <p class="card-subtitle ">
                       <%= link_to friend.name, user_path(friend) %>
                     </p>
-                    <%= form_for((current_user.active_friendships.find_by(active_friend_id: friend.id) || current_user.passive_friendships.find_by(passive_friend_id: friend.id)), html: { method: :delete }) do |f| %>
-                      <%= hidden_field_tag :active_friend_id, friend.id %>
+                    <%= form_for((current_user.passive_friendships.find_by(passive_friend_id: friend.id)), html: { method: :delete }) do |f| %>
```

Comment:
**[REQUIRED]**
* Let's separate code logic from the views([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright).